rust/migrate-wicked/src/migrate.rs

```diff
@@ -50,6 +50,13 @@ impl Adapter for WickedAdapter {
 pub async fn migrate(paths: Vec<String>) -> Result<(), Box<dyn Error>> {
     let wicked = WickedAdapter::new(paths);
     let state = wicked.read()?;
+    let settings = MIGRATION_SETTINGS.get().unwrap();
+    if settings.dry_run {
+        for connection in state.connections {
+            println!("{:?}", connection);
```

Comment:
I would like to have a pretty print option, wdyt?
```suggestion
            println!("{:#?}", connection);
```

Reply:
That output can get incredibly long and actually hide e.g. warnings from sight without scrolling up a lot which is way more important in this case IMO. This just prints the agama model which is highly likely to be pretty meaningless to users since it doesn't even represent the NM config or the agama "settings". Maybe it makes sense to even skip this printing all together. And introducing an extra option to control this seems like overkill for this to me.

Reply:
Hm, I like to see this for debugging purposes.  Let's add it to DEBUG log level, or?

Reply:
thx for adding it to debug